internal/apps/config/appdev.go

```diff
@@ -0,0 +1,223 @@
+package config
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"time"
+
+	"github.com/spf13/viper"
+)
+
+const (
+	// DefaultDevConfigFile is the name of the default dev configuration file.
+	DefaultDevConfigFile = "dev-config.yaml"
+
+	// nsComponents is the namespace of the component-specific config tree.
+	nsComponents = "components"
+)
+
+// type appDevUnknownKeyErr struct {
+// 	key string
+// }
+
+// func (e *appDevUnknownKeyErr) Error() string {
+// 	return fmt.Sprintf("unknown key: %s\nvalid keys: %s", e.key, ValidAppDevKeys())
+// }
+
+// var validAppDevKeys = map[string]bool{
```

Comment:
Are we planning on bringing this concept back? If not we should remove the commented code.

Reply:
I'm not sure, I left it in so we can decide without unblocking these changes